src/vma/main.cpp

```diff
@@ -1283,9 +1285,11 @@ void get_env_params()
 	if ((env_ptr = getenv(SYS_VAR_TIMER_RESOLUTION_MSEC)) != NULL)
 			safe_mce_sys().timer_resolution_msec = atoi(env_ptr);
 
-	if ((env_ptr = getenv(SYS_VAR_TCP_TIMER_RESOLUTION_MSEC)) != NULL) {
+	if ((env_ptr = getenv(SYS_VAR_TCP_TIMER_RESOLUTION_MSEC)) != NULL)
 			safe_mce_sys().tcp_timer_resolution_msec = atoi(env_ptr);
-	}
+
+	if ((env_ptr = getenv(SYS_VAR_INTERNAL_THREAD_TCP_TIMER_HANDLING)) != NULL)
+			safe_mce_sys().internal_thread_tcp_timer_handling = (internal_thread_tcp_timer_handling_t)atoi(env_ptr);
 
```

Comment:
I would like to suggest using strtol() instead of atoi()
strtol()
```
If no valid conversion could be performed, a zero value is returned (0L).
```
atoi()
```
If the converted value would be out of the range of representable values by an int, it causes undefined behavior.
```